web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -24,6 +24,97 @@
                 />
             </div>
         </div>
+        <div v-if="isNewAccessGrantFlow" class="access-grants__new-title-area">
+            <h2 class="access-grants__title-area__title" aria-roledescription="title">Access Management</h2>
+            <div class="access-grants__title-area__title-subtext" aria-roledescription="title">Create encryption keys to setup permissions to access your objects.</div>
+           
+        </div>
+        <div v-if="isNewAccessGrantFlow" class="access-grants__flows-area">
+            <div class="access-grants__flows-area__access-grant">
+                <div class="access-grants__flows-area__icon-container">
+                    <AccessGrantsIcon />
+                </div>
+                <!-- <div style="height: 40px; width: 40px; border: 2px solid black; margin-top: -5px">Icon</div> -->
+                <div class="access-grants__flows-area__title">Access Grant</div>
+                <div class="access-grants__flows-area__summary">Gives access through native clients such as uplink, libuplink, associate libraries, and bindings. </div>
+                <div class="access-grants__flows-area__button-container">
+                    <!-- <button class="access-grants__flows-area__learn-button">Learn More</button> -->
+                    <!-- <button class="access-grants__flows-area__create-button">Create Access Grant</button> -->
+                    <VButton
+                        label="Learn More"
+                        width="auto"
+                        height="30px"
+                        isTransparent="true"
+                        fontSize="13px"
+                        class="access-grants__flows-area__learn-button"
+                    />
+                    <VButton
+                        label="Create Access Grant"
+                        fontSize="13px"
+                        width="auto"
+                        height="30px"
+                        class="access-grants__flows-area__create-button"
+                    />
+                </div>
+            </div>
+            <div class="access-grants__flows-area__s3-credentials">
+                <div class="access-grants__flows-area__icon-container">
+                    <S3Icon />
+                </div>
+                <!-- <div style="height: 40px; width: 40px; border: 2px solid black; margin-top: -5px">Icon</div> -->
+                <div class="access-grants__flows-area__title">S3 Credentials</div>
+                <div class="access-grants__flows-area__summary">Gives access through S3 compatible tools and services via our hosted Gateway MT.</div>
+                <br/>
+                <div class="access-grants__flows-area__button-container">
+                    <!-- <button class="access-grants__flows-area__learn-button">Learn More</button> -->
```

Comment:
there are a few elements that are commented out like this. Did you intend to leave them in?

Reply:
addressed in previous commit